migration/mig_0016_add_recommendations_table.go

```diff
@@ -43,8 +43,8 @@ var mig0016AddRecommendationsTable = Migration{
 				AS SELECT
 					org_id,
 					cluster_id,
-					REGEXP_REPLACE(rule_fqdn, 'report$', error_key) AS rule_fqdn,
-					error_key
+					REGEXP_REPLACE(rule_fqdn, '.report$', CONCAT('|', error_key)) AS rule_fqdn,
```

Comment:
IIRC the `error_key` should not be appended, just store `rule_fqdn` as is (IMHO)

Reply:
I'm removing this change from the PR, as fixing this removes the sense of mig_0019